testing-modules/testng_command_line/src/main/java/com/baeldung/testing_modules/testng_command_line/DateSerializerService.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.testing_modules.testng_command_line;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateSerializerService {
+    private SimpleDateFormat simpleDateFormat;
+
+    public DateSerializerService() {
+        super();
+        simpleDateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss.SSS");
```

Comment:
Two points here.

1. This could be in the initializer list of the class, rather than in a constructor.
2. This isn't a threadsafe approach, and your tests talk about threadsafety - are you sure you want to do this?